src/variables.css

```diff
@@ -0,0 +1,6 @@
+:root {
```

Comment:
Added these CSS vars to allow using the same colours with the same names that we do in forsite-core.

I chose not to add all colours since they aren't all needed yet